students/joejohnsto/lesson03/mailroom.py

```diff
@@ -7,40 +7,99 @@
 Mailroom Part I
 """
 
+import sys
 
 donor_db = [('Bill Gates', [4000.32, 35.00, 17899.99]),
             ('Oprah Winfrey', [9999.99, 9999.99]),
             ('Rob Schneider', [400.00, 55.00, 800.00]),
             ('Donald Trump', [0.32, 5.00]),
             ('Denise Richards', [6040.77])]
 
-prompt = "\n".join(("Welcome to the mailroom!",
+prompt = "\n".join(("\nWelcome to the mailroom!",
           "Please choose from below options:",
           "1 - Send a Thank You",
           "2 - Create a Report",
           "3 - quit",
           ">>> "))
 
 
+def sort_key(donor):
+    return sum(donor[1])
+
+
 def thank_you():
+    """Add a new donation to the database and print a thank you email for the donor"""
+    
     response = 'list'
+    # get donor name from user, display donor list if asked
     while response == 'list':
-        response = input('What is the full name of the donor to whom you would like to\
-                     send a thank you?\nType "list" for a list of current donors.\n')
+        response = input('\nWhat is the full name of the donor to whom you would like to ' + \
+                     'send a thank you?\nType "list" for a list of current donors ' + \
+                     'or "q" to quit back to main menu.\n>>> ')
         if response == 'list':
             for x, y in donor_db:
                 print(x)
-        
-    name = response
-    if not [x for x, y in donor_db if x == name.title()]:
-        donor_db.append((name.title(), []))
-
-    amount = input(f'Enter the amount that {name.title()} donated: ')
-    
-    idx = [x for x, y in donor_db].index(name.title())
+        elif response == 'q':
```

Comment:
Problems with indentation

Reply:
Whats wrong with the indentation? The 'elif' is in line with the corresponding 'if' above and 'else' below.